fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -1381,6 +1385,73 @@ private void handleShowLoad() throws AnalysisException {
+        ShowCreateRoutineLoadStmt showCreateRoutineLoadStmt = (ShowCreateRoutineLoadStmt) stmt;
+        List<List<String>> rows = Lists.newArrayList();
+        List<RoutineLoadJob> routineLoadJobList;
+        try {
+            routineLoadJobList = GlobalStateMgr.getCurrentState().getRoutineLoadMgr()
+                    .getJob(showCreateRoutineLoadStmt.getDbFullName(),
+                            showCreateRoutineLoadStmt.getName(),
+                            false);
+        } catch (MetaNotFoundException e) {
+            LOG.warn(e.getMessage(), e);
+            throw new AnalysisException(e.getMessage());
+        }
+        if (routineLoadJobList == null || routineLoadJobList.size() == 0) {
+            resultSet = new ShowResultSet(showCreateRoutineLoadStmt.getMetaData(), rows);
+            return;
+        }
+        RoutineLoadJob routineLoadJob = routineLoadJobList.get(0);
+        if (routineLoadJob.getDataSourceTypeName().equals("PULSAR")) {
+            throw new AnalysisException("not support pulsar datasource");
+        }
+        StringBuilder createRoutineLoadSql = new StringBuilder();
+        try {
+            String dbName = routineLoadJob.getDbFullName();
+            createRoutineLoadSql.append("CREATE ROUTINE LOAD ").append(dbName).append(".")
+                    .append(showCreateRoutineLoadStmt.getName())
+                    .append(" on ").append(routineLoadJob.getTableName());
+        } catch (MetaNotFoundException e) {
+            LOG.warn(e.getMessage(), e);
+            throw new AnalysisException(e.getMessage());
+        }
+
+        if (routineLoadJob.getColumnSeparator() != null) {
+            createRoutineLoadSql.append("\n COLUMNS TERMINATED BY ")
+                    .append(routineLoadJob.getColumnSeparator().toSql(true));
+        }
+
+        if (routineLoadJob.getColumnDescs() != null) {
+            createRoutineLoadSql.append(",\nCOLUMNS (");
+            List<ImportColumnDesc> descs = routineLoadJob.getColumnDescs();
+            for (int i = 0; i < descs.size(); i++) {
+                ImportColumnDesc desc = descs.get(i);
+                createRoutineLoadSql.append(desc.toString());
+                if (descs.size() == 1 || i == descs.size() - 1) {
+                    createRoutineLoadSql.append(")");
+                } else {
+                    createRoutineLoadSql.append(", ");
+                }
+            }
+        }
+        if (routineLoadJob.getPartitions() != null) {
+            createRoutineLoadSql.append(",\n");
+            createRoutineLoadSql.append(routineLoadJob.getPartitions().toString());
+        }
+        if (routineLoadJob.getWhereExpr() != null) {
+            createRoutineLoadSql.append(",\nWHERE ");
+            createRoutineLoadSql.append(routineLoadJob.getWhereExpr().toSql());
+        }
+
+        createRoutineLoadSql.append("\nPROPERTIES\n").append(routineLoadJob.jobPropertiesToSql());
```

Comment:
If there are some secret message in the property, how to show them?

Reply:
@chaoyli 
![image](https://github.com/StarRocks/starrocks/assets/45813655/c45bd6f8-9686-4f9a-9c01-5fb204d6f268)
Like this.